www-client/elinks/elinks-0.13_p20180225.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Please use a version that won't be newer than the next upstream release. `0.13_pre...` would normally be a good choice. However, given that upstream has made `_pre` releases in the past, you may go even further and make it e.g. `_pre_pre...`.